src/components/staking/LiValidator.vue

```diff
@@ -36,7 +36,7 @@
       />
       <div class="validator-info">
         <h3 class="li-validator-name">
-          {{ validator.shortenName }}
+          {{ validator.name | truncate(30) }}
```

Comment:
instead of truncate please use the address shortening format we already use everywhere else like in the Bech32 component

Reply:
Oops I should have read this earlier xD

Reply:
Alright, this in now implemented